mongo/integration/unified/event.go

```diff
@@ -61,6 +67,7 @@ func monitoringEventTypeFromString(eventStr string) (monitoringEventType, bool)
 }
 
 func monitoringEventTypeFromPoolEvent(evt *event.PoolEvent) monitoringEventType {
+	// TODO GODRIVER-1827: add storePoolReady and storeConnectionCheckOutStarted events
```

Comment:
Why not add these now? Will there be more work than adding new enum variants?

Reply:
I can, but I feel like it's misleading to add them to the public events package enum before they actually do anything